addon/components/document-details.hbs

```diff
@@ -196,6 +196,8 @@
                   <a
                     uk-icon="download"
                     class="uk-icon-link"
+                    target="_blank"
+                    rel="noopener noreferrer"
```

Comment:
AFAIK the usage of `_blank` is widely discouraged ([depends on the reason](https://css-tricks.com/use-target_blank/#a-bad-reason-my-client-wants-it-that-way)). If this is for downloading a file (which I assume because of the `href`) we should use [`file-saver`](https://www.npmjs.com/package/file-saver) like in our other projects - it's the best UX.. 

CC @czosel

Reply:
Agreed 👍

Reply:
Using FileSaver triggers a HEAD request which is not allowed by Min.IO (https://github.com/eligrey/FileSaver.js/issues/557).

It still works as expected. So you can decide what you prefer.

Reply:
A HEAD request only happens when it's a CORS request. Nothing to do with file-saver.

Reply:
Do you know a way to disable the XHR sent by FileSaver?

Reply:
No. This is not an issue of the client but the server..

Reply:
cc @winged